prog/probe.go

```diff
@@ -46,25 +46,46 @@ var (
 	dockerEndpoint   = "unix:///var/run/docker.sock"
 )
 
-func check(flags map[string]string) {
```

Comment:
That includes the other refactorings in this PR. Please create a different PR if necessary.